Tests/SwiftJWTTests/TestJWT.swift

```diff
@@ -203,11 +235,108 @@ class TestJWT: XCTestCase {
                 XCTFail("Failed to decode")
             }
         }
-        else {
-            XCTFail("Failed to sign")
+    }
+
+    func testJWTEncoder() {
+        var jwt = JWT(claims: TestClaims())
+        jwt.claims.sub = "1234567890"
+        jwt.claims.name = "John Doe"
+        jwt.claims.admin = true
+        do {
+            let jwtString = try rsaJWTEncoder.encodeToString(jwt)
+            let decodedJWTString = try JWT<TestClaims>(jwtString: jwtString)
+            let decodedTestClaimJWT = try JWT<TestClaims>(jwtString: encodedTestClaimJWT)
+            jwt.header.alg = "RS256"
```

Comment:
It would be worth a comment here to explain why you are setting this field, ie. so that the JWT you just created has the same value for `alg` as the decoded one (as the `alg` header is normally determined by the signer).